src/server/game/Handlers/DuelHandler.cpp

```diff
@@ -35,9 +35,14 @@ void WorldSession::HandleDuelAcceptedOpcode(WorldPacket& recvPacket)
     sLog->outStaticDebug("Player 2 is: %u (%s)", plTarget->GetGUIDLow(), plTarget->GetName().c_str());
 #endif
 
+   if(player->duel->underDuel || plTarget->duel->underDuel)
```

Comment:
if (player->duel->underDuel || plTarget->duel->underDuel)

Reply:
?????

Reply:
if() and if () codestyle

Reply:
missing space between `if` and `(player->duel->underDuel || plTarget->duel->underDuel)`

Reply:
Done

Reply:
ah ok, you can click on "suggest" icon next time @frytik so we can see better